test/check-application

```diff
@@ -3277,6 +3278,111 @@ PodName={name}
     def testQuadletsUsers(self):
         self._testQuadlets(system=False)
 
+    def testOtherUsers(self):
+        m = self.machine
+        b = self.browser
+
+        # create two system users with running container
+        ssh = {}
+        uids = {}
+        with open(DEFAULT_IDENTITY_PUB_FILE) as f:
+            authorized_keys = f.read()
```

Comment:
Interesting, sounds like we can make the normal admin user session better as well:
```
        # Create admin session
        m.execute("""
            if [ ! -d /home/admin/.ssh ]; then
                mkdir /home/admin/.ssh
                cp /root/.ssh/* /home/admin/.ssh
                chown -R admin:admin /home/admin/.ssh
                chmod -R go-wx /home/admin/.ssh
            fi
            """)
```

I much prefer your approach with `m.write()`

Reply:
Sure! See #2062 (not ready yet, but once it is I'll ask for your review)

Reply:
#2062 landed.